blackdoc/tests/test_rst.py

```diff
@@ -31,6 +31,21 @@
         ),
         pytest.param(data.lines[38:47], None, id="ipython-prompt"),
         pytest.param(data.lines[52:64], None, id="ipython-prompt-cell-decorator"),
+        pytest.param(
+            data.lines[73:79],
+            ((1, 7), rst.name, "\n".join(data.lines[73:79])),
+            id="testsetup",
+        ),
+        pytest.param(
+            data.lines[80:83],
+            ((1, 4), rst.name, "\n".join(data.lines[80:83])),
+            id="testcode",
+        ),
+        pytest.param(
+            data.lines[84:87],
+            ((1, 4), rst.name, "\n".join(data.lines[84:87])),
+            id="testcleanup",
+        ),
```

Comment:
thanks for working through this. I really need to come up with a simpler way to test the formatting functions.